backend/controllers/userControllers.js

```diff
@@ -0,0 +1,66 @@
+//Author: Cleveland Plonsey
+//date: 11/9/2023
+//Purpose: controllers for user requests
+const connection = require("../db/connection");
+const bcrypt = require("bcrypt");
+
+const register = async (request, response) => {
+  const { username, password, email } = request.body;
+  if (username && password && email) {
+    const salt = "batteryAnt"; //TODO find out what this is for...
+    const imageUrl = "/";
+    //TODO: check if this username is availabe in the db
+    let q = "SELECT * FROM users WHERE username = $1 OR email = $2";
```

Comment:
I would use a better variable name than `q`. It does not really tell you anything about what holds. Also, if you ever want to search the entire project for SQL queries, you likely wont get much help searching the entire project for `q`

Reply:
q is the variable name convention I learned for database query string. Should I rename it queryString or postgress or something?